task4-1/binaryAddition.c

```diff
@@ -0,0 +1,59 @@
+#include <stdio.h>
+#include "binaryAddition.h"
+
+#define INT_SIZE 32
```

Comment:
```suggestion
#define INT_SIZE (sizeof(int) * 8)
```
Согласно стандарту, размер int определяется реализацией, никто не обещает, что он равен 32.